guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,268 @@
+   mobile = "/assets/images/guide/how-it-works/ecash/ecash-protocol-logos@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/ecash-protocol-logos@2x.jpg"
+   alt-text = "An illustration showing the Cashu and Fedimint protocol logos."
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+**Fedimint** and **Cashu** are two unqiue bitcoin-backed ecash protocols. They differ in their approach to trust and governance. 
+
+Fedimint operates through a federated model where a group of trusted entities (called "guardians") manage funds collectively. 
+
+Cashu lets users run and operate their own solo mints or place trust in a solo mint they choose to use.
+
+---
+
+### [Cashu]({{ '/guide/how-it-works/ecash/cashu/' | relative_url }})
+
+ Learn about Cashu's approach to bitcoin-backed ecash through the use of solo mints.
+
+---
+
+### [Fedimint]({{ '/guide/how-it-works/ecash/fedimint' | relative_url }})
+
+Learn about Fedimint's federated model, which decentralizes trust by involving multiple parties through the use of multi-sig.
+
+---
+
+## Using Ecash
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/using-ecash@1x.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/using-ecashjpg.png"
+   alt-text = "An illustration showing distorted physical paper cash along with the words USING ECASH."
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+### Creating and Redeeming Ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   alt-text = "An illustration of the process of creating and redeeming ecash tokens"
+   caption = "Creating and redeeming ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Cashu and Fedimint use various terms for the processes of creating, redeeming, and transferring bitcoin-backed ecash. To provide clarity, we'll use neutral terms in our explanations and provide a translation table for specific implementations.
+
+#### Creating Ecash
+
+**1. User Deposits Funds** - The user deposits bitcoin into a mint or federation. Typically this is done by generating a Lightning invoice through the mint and paying it.
+
+**2. Token Creation** - Upon successful payment, the wallet generates secrets and blinds them. The wallet then sends the blinded messages to the mint or federation, which returns a blind signature on the blinded messages. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
```

Comment:
```suggestion
**2. Token Creation** - Upon successful payment, the wallet generates secrets and blinds them. The wallet then sends the blinded messages to the mint or federation, which returns blind signatures on the blinded messages. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
```

To me the other wording implied it was one signature across all the blinded messages where it is one for each.

Reply:
Thanks! Just updated.

Reply:
For point 3. What do you think about:

`**3. Token receipt** - The user receives a set of proofs that correspond to specific amounts of bitcoin. These proofs can be combined to create an ecash token of any amount denominated in bitcoin.`

I'm not sure about the labeling of this step. "Token receipt". I don't think the user receives a token in this process. Is it correct to say they receive a set of proofs?